### PR TITLE
Sieve Of Eratosthenes #2174

### LINKED ISSUE #2174: Sieve Of Eratosthenes
- [x] C
- [x] C++
- [ ] C# - @kashika3101
- [x] CoffeeScript
- [ ] Dart - @shivang2k
- [x] Go
- [x] Java
- [x] JS
- [x] Kotlin
- [x] Php - @mruna7
- [x] Python
- [x] Readme
- [ ] Ruby - @Basilleaf24
- [ ] Rust
- [x] TypeScript - @shivamgupta57121

### PR DESCRIPTION
**Pull request by mruna7**
Fixes #2174 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ]  Input is taken dynamically.
- [x]  Sample Input / Output is added at the end of file.
- [x]  Logic Documentation (Comments).
- [x]  File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added sieve in PHP

#### Languages Used:
- PHP

#### Files Added:
- Sieve _Of _Eratosthenes .php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by mruna7** [comment]
@jainaman224 ,
@shreyakapoor08 
@asha15  please review this request

File: Number_Conversion/Java/Binary_To_Octal.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import java.io.*;
```
Comment (asha15) [inline Number_Conversion/Java/Binary_To_Octal.java:1]: please remove this file

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+function Sieve_of_Eratosthenes($num)
+{
+    $sieve = array_fill(0, $num+1, true); 
+    for ($i = 2; $i*$i <= $num; $i++)
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php:5]: Space around *

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+function Sieve_of_Eratosthenes($num)
+{
+    $sieve = array_fill(0, $num+1, true);
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php:4]: Space around +

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<?php
+function Sieve_of_Eratosthenes($num)
+{
+    $sieve = array_fill(0, $num+1, true); 
+    for ($i = 2; $i*$i <= $num; $i++) 
+    { 
+        if ($sieve[$i] == true) 
+        { 
+             for ($j = $i*$i; $j <= $num; $j += $i)
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php:9]: Space around *

File: Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+         if ($sieve[$i]) 
+             echo $i." "; 
+     }      
+} 
+$num = (int)readline('Enter an integer: '); 
+echo "Prime numbers upto " .$num."\n" ; 
+Sieve_of_Eratosthenes($num); 
+?>
+
+//Input:
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_Of_Eratosthenes.php:24]: Convert input and output to multi line comment

**Comment by mruna7** [comment]
> Looks good @mruna7. Please squash your commits.

@singh-shreya6  Mam,done!!

**Comment by mruna7** [comment]
> > Looks good @mruna7. Please squash your commits.

yes mam @singh-shreya6

**Comment by singh-shreya6** [comment]
Squashing is not done properly @mruna7. Please remove the extra file in files changes. Binary to Octal one.

**Comment by mruna7** [comment]
> Squashing is not done properly @mruna7. Please remove the extra file in files changed. Binary to Octal one.

Mam can you please guide me how to do that.

File: Number_Conversion/Java/Binary_To_Octal.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import java.io.*; 
+import java.util.Scanner;
```
Comment (singh-shreya6) [inline Number_Conversion/Java/Binary_To_Octal.java:2]: Please remove this file first.

**Comment by singh-shreya6** [comment]
Are you squashing the commits @mruna7 ?

**Comment by mruna7** [comment]
> Are you squashing the commits @mruna7 ?

yes i am

**Comment by mruna7** [comment]
@singh-shreya6  @asha15  please review this

**Pull request merged**